ktor-utils/common/src/io/ktor/util/collections/CollectionUtils.kt

```diff
@@ -6,7 +6,7 @@ package io.ktor.util.collections
 
 import io.ktor.util.*
 
-internal fun <T> sharedListOf(vararg values: T): MutableList<T> {
+public fun <T> sharedListOf(vararg values: T): MutableList<T> {
```

Comment:
may be mark with `InternalApi`?

Reply:
@e5l ping

Reply:
Oh, yep. Almost missed this. I'm thinking about dropping `@InternalAPI` at all. Could you tell me why it should be marked?

Reply:
mainly, because, when there will be kotlin 1.x (1.6.x - 1.7.x) with new native MM enabled by default, those functions will be not needed. So marking them with `InternalAPI` or some other annotation or may be even `Deprecated` annotation just to give some info to users on what those functions needed for.

but, as we have already `shared` delegates `public`, for me it's ok even without annotation. when new MM appear, we can just deprecate them

Reply:
Yep, that was the original plan. When the new MM will be default, we start deprecation cycle